pysatNASA/instruments/timed_guvi.py

```diff
@@ -47,108 +69,74 @@
 from pysat import logger
 
 from pysatNASA.instruments.methods import cdaweb as cdw
+from pysatNASA.instruments.methods import general as mm_nasa
+from pysatNASA.instruments.methods import jhuapl
+from pysatNASA.instruments.methods import timed as mm_timed
 
 # ----------------------------------------------------------------------------
 # Instrument attributes
 
 platform = 'timed'
 name = 'guvi'
-tags = {'sdr-imaging': 'Level 1C imaging data',
+tags = {'edr-aur': 'Level 2 Auroral disk imaging mode',
+        'sdr-imaging': 'Level 1C imaging data',
         'sdr-spectrograph': 'Level 1C spectrograph data'}
-inst_ids = {'high_res': list(tags.keys()),
-            'low_res': list(tags.keys())}
+inst_ids = {'': ['edr-aur'],
+            'high_res': ['sdr-imaging', 'sdr-spectrograph'],
+            'low_res': ['sdr-imaging', 'sdr-spectrograph']}
 
 pandas_format = False
+multi_file_day = True  # This is only true for EDR-AUR
 
 # ----------------------------------------------------------------------------
 # Instrument test attributes
 
-_test_dates = {
-    'high_res': {tag: dt.datetime(2005, 6, 28) for tag in tags.keys()},
-    'low_res': {tag: dt.datetime(2005, 6, 28) for tag in tags.keys()}}
+_test_dates = {iid: {tag: dt.datetime(2005, 6, 28) for tag in inst_ids[iid]
+               for iid in inst_ids.keys()}
```

Comment:
This is the source of the SyntaxError, which points to the following lines as the problem.

Reply:
I don't have more time to work on this today, go ahead and push if you made the changes already 🦦

Reply:
I've pushed the changes.  flake 8 passes locally, but there's still a download error.  Not sure if this is unique to me or if we'll see it on github actions too